Polyhedron/include/CGAL/boost/graph/properties_Polyhedron_3.h

```diff
@@ -484,8 +532,93 @@ struct graph_has_property<CGAL::Polyhedron_3<Gt, I, HDS, A>, vertex_index_t>
     >
 {};
 
+
+template <class Gt, class I, CGAL_HDS_PARAM_, class A, typename T>
+struct property_map<CGAL::Polyhedron_3<Gt, I, HDS, A>, boost::vertex_property_t<T> >
+{
+  typedef CGAL::Polyhedron_3<Gt, I, HDS, A> SM;
+  typedef typename boost::graph_traits<SM>::vertex_descriptor vertex_descriptor;
+  typedef CGAL::internal::Dynamic_polyhedron_property_map<SM,vertex_descriptor,T> type;
+  typedef type const_type;
+};
+
+template <class Gt, class I, CGAL_HDS_PARAM_, class A, typename T>
+struct property_map<CGAL::Polyhedron_3<Gt, I, HDS, A>, boost::halfedge_property_t<T> >
+{
+  typedef CGAL::Polyhedron_3<Gt, I, HDS, A> SM;
+  typedef typename boost::graph_traits<SM>::halfedge_descriptor halfedge_descriptor;
+  typedef CGAL::internal::Dynamic_polyhedron_property_map<SM,halfedge_descriptor,T> type;
+  typedef type const_type;
+};
+
+template <class Gt, class I, CGAL_HDS_PARAM_, class A, typename T>
+struct property_map<CGAL::Polyhedron_3<Gt, I, HDS, A>, boost::edge_property_t<T> >
+{
+  typedef CGAL::Polyhedron_3<Gt, I, HDS, A> SM;
+  typedef typename boost::graph_traits<SM>::edge_descriptor edge_descriptor;
+  typedef CGAL::internal::Dynamic_polyhedron_property_map<SM,edge_descriptor,T> type;
+  typedef type const_type;
+};
+
+template <class Gt, class I, CGAL_HDS_PARAM_, class A, typename T>
+struct property_map<CGAL::Polyhedron_3<Gt, I, HDS, A>, boost::face_property_t<T> >
+{
+  typedef CGAL::Polyhedron_3<Gt, I, HDS, A> SM;
+  typedef typename boost::graph_traits<SM>::face_descriptor face_descriptor;
+  typedef CGAL::internal::Dynamic_polyhedron_property_map<SM,face_descriptor,T> type;
+  typedef type const_type;
+};
+
+
 } // namespace boost
 
+namespace CGAL {
+
```

Comment:
again if it is an implementation details, the new functions should not pollute the CGAL namespace and should be in an internal one.